tap_linkedin_ads/schemas/creatives.json

```diff
@@ -17,183 +29,99 @@
         "integer"
       ]
     },
-    "change_audit_stamps": {
+    "content": {
       "type": [
         "null",
         "object"
       ],
       "additionalProperties": false,
       "properties": {
-        "created": {
+        "reference": {
           "type": [
             "null",
-            "object"
-          ],
-          "additionalProperties": false,
-          "properties": {
-            "time": {
-              "type": [
-                "null",
-                "string"
-              ],
-              "format": "date-time"
-            }
-          }
+            "string"
+          ]
         },
-        "last_modified": {
+        "text_ad": {
           "type": [
             "null",
             "object"
           ],
           "additionalProperties": false,
           "properties": {
-            "time": {
+            "headline": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "description": {
               "type": [
                 "null",
                 "string"
-              ],
-              "format": "date-time"
+              ]
+            },
+            "landing_page": {
+              "type": [
+                "null",
+                "string"
+              ]
             }
           }
         }
       }
     },
-    "created_time": {
+    "created_at": {
       "type": [
         "null",
         "string"
       ],
       "format": "date-time"
     },
-    "last_modified_time": {
+    "created_by": {
       "type": [
         "null",
         "string"
-      ],
-      "format": "date-time"
-    },
-    "id": {
-      "type": [
-        "null",
-        "integer"
       ]
     },
-    "processing_state": {
+    "last_modified_at": {
       "type": [
         "null",
         "string"
-      ]
+      ],
+      "format": "date-time"
```

Comment:
Seems like all date-time fields are coming in `epoch` format. Are we transforming them in code or is it not necessary to do so ?